src/content/docs/mobile-monitoring/new-relic-monitoring-react-native/react-native-installation.mdx

```diff
@@ -0,0 +1,312 @@
+```sh
+# yarn add @bibabovn/react-native-newrelic
+```
+2. Run the following:
+```shell
+  npx pod-install
+```
+  </Collapser>
+</CollapserGroup>
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id=""
+    title="Preliminary setup"
+  >
+To set up the React Native agent, you need to start it with `index.js`. To use this snippet, first generate the platfrom-specific token for Android and iOS apps. When you have the token, insert it in <var>GENERATED_TOKEN</var>:
+
+```js
+NewRelic.startAgent("<var>GENERATED_TOKEN</var>");
+NewRelic.setJSAppVersion("APP_JS_VERSION");
+
+```
+  </Collapser>
+</CollapserGroup>
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id=""
+    title="Android Setup"
+  >
+Complete the following:
+
+1. Install the New Relic native Android agent using these [instructions ](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-android-apps-gradle-android-studio).
+2. Update `build.gradle`:
+  ```java
+    buildscript {
+      ...
+      repositories {
+        ...
+        mavenCentral()
+      }
+      dependencies {
+        ...
+        classpath "com.newrelic.agent.android:agent-gradle-plugin:6.4.1"
+      }
+    }
+  ```
+
+3. Update `app/build.gradle`:
+  ```
+    apply plugin: "com.android.application"
+    apply plugin: 'newrelic' // <-- add this
+  ```
+
+4. Make sure your app requests `INTERNET` and `ACCESS_NETWORK_STATE` permissions by adding these lines to your `AndroidManifest.xml`:
+  ```
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+  ```
+  </Collapser>
+</CollapserGroup>
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id=""
+    title="iOS setup"
+  >
+
+Install the New Relic XCFramework agent:
+```shell
+  npx pod-install
+```
+
+  </Collapser>
+</CollapserGroup>
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id=""
+    title="Expo integration"
+  >
+
+Integration with Expo is possible in both bare workflow and [custom managed workflow](https://docs.expo.io/workflow/customizing/) via [config plugins](https://docs.expo.io/guides/config-plugins/).
+
+* [Bare workflow](https://docs.expo.dev/introduction/managed-vs-bare/#bare-workflow): Please follow the above installation steps instead.
+* [Managed workflow](https://docs.expo.dev/introduction/managed-vs-bare/#bare-workflow): After installing our package, add the config plugin to the plugins array of your `app.json` or `app.config.js`.
+
+After this, you need to use the `expo prebuild --clean` command as described in the ["Adding custom native code"](https://docs.expo.dev/workflow/customizing/) guide to rebuild your app with the plugin changes. If this command is not running, you'll get errors when starting the New Relic agent.
+  </Collapser>
+</CollapserGroup>
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id=""
+    title="Usage"
```

Comment:
I'm not exactly sure what 'usage' refers to here. If the table of calls below is exactly the same as the Android doc, I'd recommend we not include the table redundantly, and instead do something like 'You'll use the same set of API calls used by our Android agent. See ....' The less redundancy we can, the better, as sometimes people will change things in one spot and not the other and things drift out of date over time.